libnmstate/ifaces/ifaces.py

```diff
@@ -177,6 +178,25 @@ def _mark_vf_interface_as_absent_when_sriov_vf_decrease(self):
                     if vf_iface:
                         vf_iface.mark_as_absent_by_desire()
 
+    def _mark_bridge_ports_as_removed(self):
+        """
+        When a Linux bridge port is unmanaged and is removed Nmstate should
```

Comment:
The unmanaged is dedicated to NetworkManager. The `BaseIface` does not need to aware of that.
Please place the workaround code in `libnmstate/nm`.

Current code in `Ifaces` already mark removed port `BaseIface` as `is_changed` in function `Ifaces._handle_master_slave_list_change()`.

Reply:
Different solution.